docs/build-mvm/how-to/schedule_functions.md

```diff
@@ -1 +1,883 @@
+3. The spell script is located in the `spell.aqua` file:
+    
+    ```aqua
+    -- the Fluence CLI version is 0.13.5
+    
+    aqua MyFirstSpell
+    
+    -- Note: spell main function must be exported
+    export spell
+    
+    import Spell from "@fluencelabs/spell/spell_service.aqua" -- (1)
+    
+    func spell(): -- (2)
+        Spell "myFirstSpell" -- (3)
+        Spell.store_log("Spell 'myFirstSpell' is working!") -- (4)
+    ```
+    
+    Let’s review the most important parts of the file:  
+    
+    1. Importing the (Spell Service API)[#how-to-resolve-your-spell-service] is required to use logging capabilities, private spell storage, etc. Note that you can implement spells without ever interacting with the Spell Service if you don’t need the features it provides.
+    2. The `spell` function here is an entry point to the spell’s execution. The main function has several features we explore in the [later](#inital-arguments) sections.
+        
+        You can modify the name of the main function to any valid name, but you also need to modify the name in the `spell.yaml` configuration file.
+        
+    3. To write logs in the [Spell Log Storage](#spell-logs-api), you need to resolve [link to the services resolution in aqua] the Spell Service that belongs to the executing spell using the Spell’s Name.
+        
+        Note that you have several options to resolve the correct spell service, which we’ll explore [later](#how-to-resolve-your-spell-service).
+        
+    4. Here, we call the function provided by the Spell Service to store the message in the logs. This message will be added to the logs each spell runs. 
+4. The spell configuration is located in the `spell.yaml` file and defines several important aspects of the spell execution. Let’s look at the content of the default spell configuration file:
+    
+    ```yaml
+    # FCLI: 0.13.5
+    version: 0
+    
+    aquaFilePath: ./spell.aqua # <-- (1)
+    
+    function: spell # <-- (2)
+    
+    clock: # <-- (3)
+      periodSec: 60
+      endDelaySec: 1800
+    ```
+    
+    1. `aquaFilePath` must be set to the aqua file with the main spell function; the path is relative to the `spell.yaml` path, so `./spell.aqua` works since it’s in the same directory.
+    2. `function` must be set to the main spell function, by default, `spell` we have seen when we look into the `spell.aqua` file. You choose any valid name you want.
+    3. `clock` defines the configuration of when and how your spell is run.
+        1. `periodSec` defines how *often* your spell is run. `periodSec: 60` means that the spell is run every minute. 
+        2. `endDelaySec`defines when the spell must be stopped from execution. `endDelaySec: 1800` means that the spell will no longer be periodically executed in 30 minutes.
+            
+            **Warning:** this delay is evaluated on the client, so the 30 minutes are counted since you created your deal, not since the spell was installed on a peer. ::
```

Comment:
What's the correct syntax for warnings?

Reply:
https://docusaurus.io/docs/next/markdown-features/admonitions

```suggestion
::: warning this delay is evaluated on the client, so the 30 minutes are counted since you created your deal, not since the spell was installed on a peer. :::
```
i think it only shows in docusaurus published docs not github readme.